src/ui.form-calendar/configs/default.config.js

```diff
@@ -14,12 +14,12 @@ export default /*tw*/ {
       },
     },
   },
-  nextPrevButton: "",
-  nextIcon: "",
+  nextPrevButton: "{UButton}",
+  nextIcon: "{UIcon}",
   nextIconName: "keyboard_arrow_right",
-  prevIcon: "",
+  prevIcon: "{UIcon}",
   prevIconName: "keyboard_arrow_left",
-  dayView: "",
+  dayView: "{DayView}",
```

Comment:
{DayView}  {MonthView} {YearView} internal components no need to be placed here.